src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private Figure[] figures = {
+            new Circle(0),
+            new IsoscelesTrapezoid(0, 0, 0),
+            new Rectangle(0, 0),
+            new Square(0),
+            new RightTriangle(0, 0)};
+
+    public Figure getRandomFigure() {
+        Figure figure = figures[random.nextInt(figures.length)];
+        return figure.createRandomFigure();
+    }
+
+    public Figure getDefaultFigure() {
+        Figure circle = new Circle(10);
+        circle.setColor(FigureColor.WHITE.toString());
+        return circle;
```

Comment:
Don't use static methods in your solution. It seems like the `createRandomFigure` method on figure might be static. Instead, create an instance of a class which method you want to call.